src/test/java/org/fundacionjala/pivotal/steps/ProjectStep.java

```diff
@@ -0,0 +1,57 @@
+package org.fundacionjala.pivotal.steps;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import org.fundacionjala.pivotal.objects.CreateProject;
+import org.fundacionjala.pivotal.objects.ProjectValues;
+import org.testng.Assert;
+
+import java.util.Map;
+
+
+/**
+ * Class Project Steps for create a project.
+ */
+public class ProjectStep {
+    private CreateProject createProject = new CreateProject();
+    private Map<ProjectValues, Object> projects;
+
+    /**
+     * Steps of Creating a Project.
+     *
+     * @param values for enter.
+     */
+    @Given("^I can create a new project with the following values$")
+    public void iCanCreateANewProjectWithTheFollowingValues(final Map<ProjectValues, Object> values) {
+        this.projects = values;
+        createProject.setButtonCreateProject();
+        projects.keySet().stream().forEach((step) -> createProject.getStrategyStepMap(projects).get(step).execute());
+        createProject.setButtonCreate();
+
+    }
+
+    /**
+     * Verify taht is created the project.
+     *
+     * @param projectName name of project.
+     */
+    @Then("^I can verify the new project with \"([^\"]*)\" project name$")
+    public void iCanVerifyTheNewProjectWithProjectName(final String projectName) {
+        Assert.assertEquals(projectName, createProject.verifyName());
+    }
+
+    /**
+     * Method for Create a new project with the following Values and exits account.
+     *
+     * @param values Map<ProjectValues, Object>.
+     */
+    @Given("^I can create a new project with the following values and exist Account$")
+    public void iCanCreateANewProjectWithTheFollowingValuesAndExistAccount(final Map<ProjectValues, Object> values) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "iCanCreateANewProjectWithTheFollowingValues" on line 25. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)